build.rs

```diff
@@ -119,12 +132,38 @@ fn main() {
   lockfile.unlock().expect("Couldn't unlock lockfile");
 }
 
+fn build_binding() {
+  let output = Command::new(python())
+    .arg("./tools/get_bindgen_args.py")
+    .arg("--gn-out")
+    .arg(build_dir().join("gn_out"))
+    .output()
+    .unwrap();
+  let args = String::from_utf8(output.stdout).unwrap();
+  let args = args.split('\0').collect::<Vec<_>>();
+
+  let bindings = bindgen::Builder::default()
+    .header("src/binding.hpp")
+    .parse_callbacks(Box::new(bindgen::CargoCallbacks::new()))
+    .clang_args(["-x", "c++", "-std=c++20", "-Iv8/include"])
+    .clang_args(args)
+    .allowlist_item("RUST_.*")
+    .generate()
+    .expect("Unable to generate bindings");
```

Comment:
Make sure this prints out the error from `bindgen`, it's gonna be painful to debug otherwise

Reply:
it gives pretty useful output
![](https://gc.gy/e91f3bec-ab4e-49c7-bb8e-c15f597d8e99.png)